client/state/login/reducer.js

```diff
@@ -48,6 +51,9 @@ export const isRequesting = createReducer( false, {
 	[ SOCIAL_LOGIN_REQUEST ]: () => true,
 	[ SOCIAL_LOGIN_REQUEST_FAILURE ]: () => false,
 	[ SOCIAL_LOGIN_REQUEST_SUCCESS ]: () => false,
+	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST ]: () => true,
+	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST_FAILURE ]: () => false,
+	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST_SUCCESS ]: () => false,
 } );
```

Comment:
I suspect we should add the `SOCIAL_CONNECT_ACCOUNT_REQUEST*` events to this reducer.

Reply:
It looks like many of our login reducers don't handle all login types which means they probably don't need that information to work properly anyway. Knowing that I'm not sure we should update, and if we do, it's best to do that in another PR imo